Systematics/test/sig_jobs_vrt_fiducial_IA_trees_compCand.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
these files (if needed at all) should go into a subdirectory (the same that is done for the STXS)